.eslintrc

```diff
@@ -53,6 +53,7 @@
     "jsx-a11y/no-static-element-interactions": 0,
     "import/prefer-default-export": 0,
     "object-curly-newline": ["error", {"ObjectPattern": { "multiline": true }}],
-    "react/require-default-props": [0]
+    "react/require-default-props": [0],
+    "jsx-a11y/label-has-for": [ 2, { "required": {"every": [ "id" ]} } ]
```

Comment:
[According to this doc page](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-for.md) this option has been deprecated in favor of [this one](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md)